packages/synapse-react-client/src/components/EntityLink.tsx

```diff
@@ -64,7 +64,21 @@ export const EntityLink = (props: EntityLinkProps) => {
     } else {
       type = getEntityTypeFromHeader(entity)
     }
-
+    if (typeof link === 'string' && link) {
+      return (
+        <Link
+          className={className}
+          target="_blank"
+          rel="noopener noreferrer"
+          href={link}
+        >
+          {showIcon && (
+            <EntityTypeIcon type={type} style={{ marginRight: '6px' }} />
+          )}
+          {entity[displayTextField as keyof typeof entity]}
+        </Link>
+      )
+    }
```

Comment:
Here you duplicated the `<Link>` return, so we have to independently re-implement all of the other features like `showIcon`, `displayTextField`, etc.

Can you refactor so that we just have one returned `Link` component, where the `href` prop is dynamically configured?